test/parallel/test-v8-inspector-json-protocol.js

```diff
@@ -0,0 +1,22 @@
+// Flags: --inspect={PORT}
```

Comment:
I had not seen this syntax before... specifically not with the `{PORT}` syntax (no other examples in test).

Is this documented anywhere? Should it be `common.PORT`?

I'm assuming I am wrong here, just curious